package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@automattic/vip-go-mu-plugins",
+  "private": true,
+  "version": "1.0.0",
+  "description": "mu-plugins used on the VIP Go platform",
+  "scripts": {
+    "phpcs": "vendor/bin/phpcs",
+    "phpcs:changed:remote": "vendor/bin/phpcs --report=bin/phpcs-eslint-report.php | eslines --quiet",
+    "phpcs:changed:local": "vendor/bin/phpcs --report=bin/phpcs-eslint-report.php | eslines --quiet -d index"
```

Comment:
What's the difference between `phpcs:changed:remote` and `:local`?

Reply:
> --diff or -d: let you choose between two diff strategies for the downgrade-unmodified-lines processor
> index: to diff HEAD against the git index.
> remote: to diff HEAD against the git remote. This is the default.

In our case, index => local and remote => remote.

We'd use `:local` for a pre-commit hook, and `:remote` for PRs / CI.

https://github.com/Automattic/eslines#runtime-options